lab-nicole/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "lab-nicole",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "DEBUG='mp3uploader*' jest mp3-router.test.js",
+    "coverage": "DEBUG='mp3uploader*' jest --coverage",
+    "start": "DEBUG='mp3uploader*' node server.js",
+    "lint": "eslint"
+  },
```

Comment:
you need to add the `"jest": { "testEnvironment": "node" }` so that your tests don't potentially hang on other people's machines.